client/.storybook/preview.tsx

```diff
@@ -0,0 +1,91 @@
+import { AuthProvider } from '../src/contexts/AuthContext'
+import { QueryClient, QueryClientProvider } from 'react-query'
+import { MemoryRouter } from 'react-router'
+import { theme } from '../src/theme'
+import { ChakraProvider } from '@chakra-ui/react'
+import * as React from 'react'
+import { initialize, mswDecorator } from 'msw-storybook-addon'
+import { rest } from 'msw'
+import { HelmetProvider } from 'react-helmet-async'
+import { MockUserData } from '../src/__mocks__/mockData'
+
+// Initialize Mock Service Worker
+initialize()
+
+const queryClient = new QueryClient()
+
+const withChakra = (StoryFn: Function) => {
+  return (
+    <ChakraProvider theme={theme}>
+      <MemoryRouter>
+        <QueryClientProvider client={queryClient}>
+          <AuthProvider>
+            <HelmetProvider>
+              <StoryFn />
+            </HelmetProvider>
+          </AuthProvider>
+        </QueryClientProvider>
+      </MemoryRouter>
+    </ChakraProvider>
+  )
+}
+
+export const decorators = [withChakra, mswDecorator]
+
+const customViewports = {
+  desktop: {
+    name: 'Desktop',
+    styles: {
+      width: '1440px',
+      height: '963px',
+    },
+  },
+  tablet: {
+    name: 'Tablet',
+    styles: {
+      width: '780px',
+      height: '801px',
+    },
+  },
+  mobile: {
+    name: 'Mobile',
+    styles: {
+      width: '470px',
+      height: '601px',
+    },
+  },
+}
```

Comment:
any way we can have the widths synced to `theme/breakpoints`?

Reply:
Will use theme/breakpoints. Use breakpoint object instead of createBreakPoints due to planned deprecation https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme-tools/src/create-breakpoints.ts